CSS/style.css

```diff
@@ -0,0 +1,217 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;600&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    height: 100vh;
+    overflow-y: scroll;
+    overflow-x: hidden;
+}
+
+.navbar {
+    width: 100%;
+    position: absolute;
+    height: 72px;
+    left: 0;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+    padding: 26px 164px 26px 155px;
+    align-items: center;
+    background: #fff;
+}
+
+.M-logo {
+    font-family: "Poppins"sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+}
+
+.nav_toggle {
+    font-family: "Poppins"sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+}
+
+#menu-list {
+    display: flex;
+    align-items: center;
+    gap: 32px;
+}
+
+.B-content {
+    display: flex;
+    justify-content: center;
+    height: 800px;
+    flex-direction: column;
+    width: 920px;
+    margin: auto;
```

Comment:
- [ ] Based on your new changes, your mobile view is far from on par with what's asked for from the figma design given. please refactor your CSS to create the dimensions given on the figma design I've attached pictures for comparison for you below. these change effects are recommended for a better user experience of your web app

## Your current view:
![image](https://user-images.githubusercontent.com/50990925/219569123-c2f77f88-d9b3-46d2-9ce6-807c375310c0.png)

## expected figma view: The black spot just shows segmented the navbar from the mainsection yours should be joined
![image](https://user-images.githubusercontent.com/50990925/219569635-2207535d-b0e9-42fb-bf90-bad294dcd6ee.png)